### PR TITLE
ChowPhaser: init at 1.1.1

### PR DESCRIPTION
**Pull request by magnetophon**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by gador** [comment]
Result of `nixpkgs-review pr 162097` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>ChowPhaser</li>
  </ul>
</details>

**Comment by gador** [comment]
##### Reviewed points

- [x] package path fits guidelines
- [x] package name fits guidelines
- [x] package version fits guidelines
- [x] package build on x86_64-linux
- [x] executables tested on ARCHITECTURE
- [x] `meta.description` is set and fits guidelines
- [x] `meta.license` fits upstream license
- [x] `meta.platforms` is set
- [x] `meta.maintainers` is set
- [x] build time only dependencies are declared in `nativeBuildInputs`
- [x] source is fetched using the appropriate function
- [x] phases are respected
- [x] patches that are remotely available are fetched with `fetchpatch`

##### Possible improvements
-

##### Comments
LGTM

**Comment by Lassulus** [comment]
not sure how to hold it:
```
[nix-shell:~/.cache/nixpkgs-review/pr-162097]$ ChowPhaserStereo
Segmentation fault (core dumped)

[nix-shell:~/.cache/nixpkgs-review/pr-162097]$ ChowPhaserMono
Segmentation fault (core dumped)
```

File: pkgs/applications/audio/ChowPhaser/default.nix [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+stdenv.mkDerivation rec {
+  pname = "ChowPhaser";
+  version = "1.1.1";
+
+  src = fetchFromGitHub {
+    owner = "jatinchowdhury18";
+    repo = "ChowPhaser";
+    rev = "v${version}";
+    sha256 = "sha256-9wo7ZFMruG3QNvlpILSvrFh/Sx6J1qnlWc8+aQyS4tQ=";
+    fetchSubmodules = true;
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/ChowPhaser/default.nix:16]:
```suggestion
    fetchSubmodules = true;
    sha256 = "sha256-9wo7ZFMruG3QNvlpILSvrFh/Sx6J1qnlWc8+aQyS4tQ=";
```
submodules influences the hash nad we usually put it last but more importantly: can we devendor those libraries?

File: pkgs/applications/audio/ChowPhaser/default.nix [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    cd ../../ChowPhaserStereo_artefacts/Release
+    cp libChowPhaserStereo_SharedCode.a  $out/lib
+    cp -r VST3/ChowPhaserStereo.vst3 $out/lib/vst3
+    cp Standalone/ChowPhaserStereo  $out/bin
+  '';
+
+  meta = with lib; {
+    homepage = "https://github.com/jatinchowdhury18/ChowPhaser";
+    description =
+      "Phaser effect based loosely on the Schulte Compact Phasing 'A'";
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/ChowPhaser/default.nix:72]:
```suggestion
    description = "Phaser effect based loosely on the Schulte Compact Phasing 'A'";
```

File: pkgs/applications/audio/ChowPhaser/default.nix (hunk 1 above)
Comment (magnetophon) [reply pkgs/applications/audio/ChowPhaser/default.nix:16]: What does devendor mean?

Comment (Lassulus) [reply pkgs/applications/audio/ChowPhaser/default.nix:16]: don't fetch the submodules but package them as separate nix packages so they can be updated, installed etc. independently. but not sure if that's possible or easy here.

**Pull request merged**